.gitignore

```diff
@@ -162,3 +162,6 @@ Icon
 Network Trash Folder
 Temporary Items
 .apdisk
+
+# This directory rebuilds on 'tox -e docs' therefore not needed in repo
+docs/api
```

Comment:
Please comment if this makes sense

Reply:
ich weiß nicht, ob da iwelche "gerüst"-dateien generiert werden, die wir vllt doch brauchen? @hf-aschloegl ?